hastory-data/src/Data/Hastory/Utils.hs

```diff
@@ -20,3 +30,25 @@ doCountsWith conv func = foldl go HM.empty
       where
         a Nothing = Just 1
         a (Just d) = Just $ d + func k
+
+dataBaseSpec :: SpecWith SqlBackend -> Spec
+dataBaseSpec = around withDatabase
+
+withDatabase :: ActionWith SqlBackend -> IO ()
+withDatabase = bracket setupDb teardownDb
+
+setupDb :: IO SqlBackend
```

Comment:
I think using [`withSqliteConn`](https://hackage.haskell.org/package/persistent-sqlite-2.10.5/docs/Database-Persist-Sqlite.html#v:withSqliteConn) would be much simpler.